core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -2741,6 +2755,18 @@ class ReplicaManager(val config: KafkaConfig,
     }
   }
 
+  private def mayUpdateTopicAssignment(partition: Partition, partitionDirectoryId: Option[Uuid]) = {
```

Comment:
did you mean `maybeUpdateTopicAssignment`?

Reply:
yup, fixed the name